tools/tcpconnect_example.txt

```diff
@@ -41,16 +41,20 @@ process to various other addresses. A few connections occur every minute.
 USAGE message:
 
 # ./tcpconnect -h
-usage: tcpconnect [-h] [-t] [-p PID]
+usage: tcpconnect [-h] [-t] [-p PID] [-P PORT [PORT ...]]
 
 Trace TCP connects
 
 optional arguments:
   -h, --help         show this help message and exit
   -t, --timestamp    include timestamp on output
   -p PID, --pid PID  trace this PID only
+  -P PORT [PORT ...], --port PORT [PORT ...]
+                     trace these ports only
 
 examples:
     ./tcpconnect           # trace all TCP connect()s
     ./tcpconnect -t        # include timestamps
     ./tcpconnect -p 181    # only trace PID 181
+    ./tcpconnect -P 80     # only trace port 80
+    ./tcpconnect -P 80 81  # only trace port 80 and 81
```

Comment:
what other tool does this? has an unquoted list as an optional argument. Is it hard to make this either:

- -P "80 81"
- -P 80,81

which I think better matches expectations? (1st is from ps(1)'s -p behavior). Plus will work when copying to other tools, where we have positional parameters.